pydantic/types.py

```diff
@@ -1094,7 +1095,7 @@ class Model(BaseModel):
 ```py
 import uuid
 
-from pydantic import BaseModel, UUID3
+from pydantic import UUID3, BaseModel
```

Comment:
I reverted this change but looks like something in our docstring formatting stuff is sorting in reverse